web/style.css

```
@@ -0,0 +1,65 @@
+
+.hint,
+.selection,
+.status {
+  opacity: 0.72;
+}
+
+.picker {
+  display: block;
+  padding: 1rem;
+  margin: 1.5rem 0 0.75rem;
+  border: 1px dashed currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)